tests/perf-test/ReportActionsList.perf-test.js

```diff
@@ -40,6 +40,9 @@ jest.mock('@react-navigation/native', () => {
     return {
         ...actualNav,
         useRoute: () => mockedNavigate,
+        useIsFocused: () => ({
+            navigate: jest.fn(),
```

Comment:
why the return value of the mocked `useIsFocused` is an object instead of a boolean value?

Reply:
I just took it from here
https://github.com/Expensify/App/blob/39a257d1aa850e875bcf604f6c7b039527234952/tests/perf-test/ReportActionCompose.perf-test.js#L32-L34

Reply:
But we can also make it as this too, tell me your preference
https://github.com/Expensify/App/blob/39a257d1aa850e875bcf604f6c7b039527234952/tests/perf-test/SelectionList.perf-test.js#L53-L54

Reply:
Changed it to return true 👍